frontend/@types/gql/queries.d.ts

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I thought we had GraphQL types in `@types/console/generated` so this shouldn't be necessary?

cc @rawagner

Reply:
Thats right. The location changed recently in https://github.com/openshift/console/pull/8343. `@types/gql` is the old location which is not ignored by git anymore so got commited here probably.

Reply:
Removed the extra files